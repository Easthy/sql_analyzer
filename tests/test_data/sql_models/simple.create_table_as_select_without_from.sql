BEGIN TRANSACTION;

CREATE TABLE IF NOT EXISTS simple.create_table_as_select_without_from AS
SELECT 1 AS user_id,
       CURRENT_DATE AS registration_date
;

END TRANSACTION;