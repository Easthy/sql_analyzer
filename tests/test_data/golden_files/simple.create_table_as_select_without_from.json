{
  "nodes": [
    { "id": "col:simple.create_table_as_select_without_from.user_id" },
    { "id": "col:simple.create_table_as_select_without_from.registration_date" },
    { "id": "tbl:simple.create_table_as_select_without_from" }
  ],
  "links": [
    { "source": "tbl:simple.create_table_as_select_without_from", "target": "col:simple.create_table_as_select_without_from.user_id", "type": "contains_column" },
    { "source": "tbl:simple.create_table_as_select_without_from", "target": "col:simple.create_table_as_select_without_from.registration_date", "type": "contains_column" }
  ]
}